style.css

```diff
@@ -26,265 +37,550 @@
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .partners img {
+    flex-wrap: wrap;
+    padding-right: 20px;
+    padding-top: 20px;
+  }
+
+  .mobile-first-page h1 {
     font-size: 40px;
-    width: 100%;
-    background-image: url(./images/font.jpg);
-    background-repeat: no-repeat;
-    background-size: cover;
-    -webkit-background-clip: text;
-    background-clip: text;
   }
-  
-  .mobile-first-page p {
-    margin-top: 20%;
-    background-color: var(--secondary_2);
-    border: 3px solid var(--secondary_1);
-    color: var(--primary_4);
-    padding: 20px;
-    line-height: 25px;
-    width: 100%;
+
+  .mobile-first-page h2 {
+    font-size: 70px;
+    max-width: 90em;
+  }
+
+  .speaker-1 h2,
+  h3,
+  h4 {
+    max-width: 10em;
   }
-  
+
   .mobile-first-page h3 {
-    font-family: 'cocogoose', sans-serif;
-    font-size: 28px;
-    font-weight: 700;
-    color: var(--primary_4);
-    margin-top: 5%;
+    max-width: 90em;
   }
-  
-  .mobile-first-page h4 {
-    font-family: 'Lato', sans-serif;
-    font-size: 23px;
-    color: var(--primary_4);
-    margin-top: 2%;
-    padding-bottom: 10%;
+
+  .mobile-first-page {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 15%;
+    margin-right: 1em;
+    margin-left: 1em;
+    background-size: 10%, 5%, 3%, 3%;
+    background-position: 90% 97%, 60% 40%, 90% 3%, 5% 50%;
   }
-  
-  .mobile-second-page {
-    display: grid;
-    gap: 20px;
-    justify-content: center;
-    align-items: center;
-    background-image: url(./images/xpattern.jpg);
-    width: 100%;
-    background-repeat: repeat;
-    background-size: 10%;
-    background-color: var(--primary_3);
-    padding-top: 5%;
+
+  .card-1,
+  .card-2,
+  .card-3,
+  .card-4,
+  .card-5 h3,
+  p {
+    text-align: center;
   }
-  
-  .mobile-second-page h2 {
-    font-family: 'Lato', sans-serif;
-    font-size: 25px;
-    font-weight: 800;
-    margin: 1em auto;
-    color: var(--secondary_1);
+
+  .mobile-first-page p {
+    margin-top: 5%;
+    background-color: #f3e7e7;
+    border: 3px solid var(--secondary_1);
+    color: var(--primary_1);
+    padding: 20px;
+    line-height: 25px;
+    width: 40%;
+    font-size: 20px;
   }
-  
-  .mobile-second-page h2::after {
-    content: "";
-    background-color: var(--primary_2);
-    display: block;
-    margin: 0 auto;
-    width: 1.5em;
-    height: 2px;
-    border-radius: 0.5px;
-    margin-top: 0.5em;
+
+  .card-box {
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
+    gap: 20px;
+    flex-wrap: wrap;
   }
-  
+
   .card-1,
   .card-2,
   .card-3,
   .card-4,
   .card-5 {
     display: flex;
+    flex-direction: column;
+    align-items: center;
     justify-content: space-between;
     background-color: rgba(252, 241, 241, 0.277);
     padding: 30px;
-    margin-left: 1em;
-    margin-right: 1em;
-    gap: 20px;
+    width: 250px;
+    margin-left: 0;
+    margin-right: 4px;
+    margin-bottom: 100px;
```

Comment:
- [ ] Some text are longer than others, that makes your cards have different heights, while the Behance design kept them all in the same size. 
- [ ] In the original design, the gap between the cards is very thin, you should consider implementing that.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142470660-b2f7cf6e-587c-43c1-9904-019ceb0a4a17.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/142470722-d2df0848-ec75-4a38-b1dc-3350115953e3.png)